tests/cpp_api_tests/rocAL_audio_unittests/rocal_audio_unittest.py

```diff
@@ -101,7 +107,7 @@ def main():
     subprocess.run(["cmake", script_path], cwd=".")   # nosec
     subprocess.run(["make", "-j16"], cwd=".")    # nosec
 
-    run_unit_test(input_file_path, qa_mode, gpu, downmix, build_folder_path, case_list)
+    run_unit_test(rocal_data_path, qa_mode, gpu, downmix, build_folder_path, case_list)
```

Comment:
@HazarathKumarM I would suggest, keep the input file_path itself, so if downmix is enabled change the input file_path variable and pass, Since user can provide custom audio path as well here